src/pages/deck-editor/deck-editor.tsx

```diff
@@ -110,6 +117,12 @@ export const DeckEditorPage = ({
   }
 
   function handleDeleteDeckClick(event: React.MouseEvent) {
-    onGoBackClick(event);
+    onGoBackClickHandler();
+  }
+
+  function onGoBackClickHandler() {
+    // todo: discuss what 'go back' should actually do
+    // can use navigate(-1) to go back a page, but this can have issues
```

Comment:
if we have trouble popping the history, we could always change the text to be a fixed page

Reply:
not really an issue, peep the side note, just potentially confusing

per your fixed page suggestion, we _could_ just send them back to the deck page, that's not a bad idea -- but maybe i'm thinking too much and just popping the history stack is good enough. thoughts?

Reply:
Yeah, I saw we have it fixed for now. We could have something like `back to decks`. Then we can always change later when we have more inward routes from other pages